packages/eslint-plugin-testing-library-react/test/rules.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { existsSync } from 'fs';
+import { resolve } from 'path';
+import * as plugin from '../src';
+
+const ruleNames = Object.keys(plugin.rules);
+const numberOfRules = 1;
+
+describe('rules', () => {
+  it('should have a corresponding doc for each rule', () => {
+    ruleNames.forEach(rule => {
+      const docPath = resolve(__dirname, '../docs/rules', `${rule}.md`);
+
+      if (!existsSync(docPath)) {
+        throw new Error(
+          `Could not find documentation file for rule "${rule}" in path "${docPath}"`
+        );
+      }
+    });
+  });
+
+  it('should have the correct amount of rules', () => {
+    const { length } = ruleNames;
+    if (length !== numberOfRules) {
+      throw new Error(
+        `There should be exactly ${numberOfRules} rules, but there are ${length}. If you've added a new rule, please update this number.`
+      );
+    }
+  });
+});
```

Comment:
Should we in the spec also throw some code at our function above?

Reply:
I guess, I have to find an example how to test the rules.